lib/traject/vatican_iiif_config.rb

```diff
@@ -24,6 +27,189 @@
+  to_field 'manuscript_additions_tesim', extract_xml('msPart/physDesc/additions/p', nil) #	Allegati	Additions
+  to_field 'manuscript_condition_tesim', extract_xml('msPart/physDesc/condition/p', nil) #	Stato di conservazione	Condition
+  to_field 'manuscript_signatures_tesim', extract_xml('msPart/physDesc/signatures', nil) #	Segnature di fascicoli	Signatures
+  to_field 'manuscript_catchwords_tesim', extract_xml('msPart/physDesc/catchwords', nil) #	Verba reclamantia	Catchwords
+  to_field 'manuscript_palimpsest_tesim', extract_xml('msPart/physDesc/palimpsest/p', nil) #	Palinsesto	Palimpsest
+  to_field 'manuscript_physical_description_tesim', extract_xml('msPart/physDesc/p', nil) #	Descrizione fisica	Physical description
+  to_field 'manuscript_heraldry_tesim', extract_xml('msPart/head/heraldry', nil) #	Stemma	Heraldry
+  to_field 'manuscript_seal_tesim', extract_xml('msPart/head/seal', nil) #	Sigillo	Seal
+  to_field 'manuscript_format_tesim', extract_xml('msPart/head/format', nil) #	Formato	Format
+  to_field 'manuscript_watermarks_tesim', extract_xml('msPart/head/watermarks/p', nil) #	Filigrane	Watermarks
+  to_field 'manuscript_motto_tesim', extract_xml('msPart/head/motto', nil) #	Motto	Motto
+  to_field 'manuscript_locus_tesim', extract_xml('msPart/msContents/msItem/locus', nil) #	Locus	Locus
+  to_field 'manuscript_author_tesim', extract_xml('msPart/msContents/msItem/author/alias/authorityAuthor[@rif="aut"]', nil) #	Autore	Author
+  to_field 'manuscript_other_author_tesim', extract_xml('msPart/msContents/msItem/name[@role="internal" or @role="external"]/alias/authorityAuthor[@rif="aut"]', nil) #	Altro autore	Other author
+  to_field 'manuscript_title_tesim', extract_xml('msPart/msContents/msItem/title[@type="title"]/@value', nil) #		Titolo	Title
+  to_field 'manuscript_supplied_title_tesim', extract_xml('msPart/msContents/msItem/title[@type="supplied"]/@value', nil) #		Titolo supplito	Supplied title
+  to_field 'manuscript_uniform_title_tesim', extract_xml('msPart/msContents/msItem/uniformTitle/alias/authorityTitleSeries[@rif="aut"]/@value', nil) #		Titolo uniforme	Uniform title
+  to_field 'manuscript_rubric_tesim', extract_xml('msPart/msContents/msItem/rubric', nil) #	Rubrica	Rubric
+  to_field 'manuscript_summary_tesim', extract_xml('msPart/msContents/msItem/summary', nil) #	Sommario	Summary
+  to_field 'manuscript_incipit_text_tesim', extract_xml('msPart/msContents/msItem/incipit[@type="text"]/@value', nil) #		Incipit testo	Incipit text
+  to_field 'manuscript_incipit_dedication_tesim', extract_xml('msPart/msContents/msItem/incipit[@type="dedication"]/@value', nil) #		Incipit dedica	Incipit dedication
+  to_field 'manuscript_incipit_preface_tesim', extract_xml('msPart/msContents/msItem/incipit[@type="preface"]/@value', nil) #		Incipit prefazione	Incipit preface
+  to_field 'manuscript_incipit_tesim', extract_xml('msPart/msContents/msItem/incipit[not(@type, nil)]/@value', nil) #		Incipit	Incipit
```

Comment:
I have a couple find+replace errors to clean up (here on 181, and below on 185.)